src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -72,32 +66,13 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
 
   useExperiences(venue.name);
 
-  const { userWithId } = useUser();
-
   const jazzbarTables = venue.config?.tables ?? JAZZBAR_TABLES;
 
   const [seatedAtTable, setSeatedAtTable] = useState("");
   const [isAudioEffectDisabled, setIsAudioEffectDisabled] = useState(false);
-
-  const dispatch = useDispatch();
-  const venueId = useVenueId();
-
-  // @debt de-duplicate this with version in src/components/templates/Audience/Audience.tsx
-  const reactionClicked = useCallback(
-    (emojiReaction: EmojiReactionType) => {
-      if (!venueId || !userWithId) return;
-
-      dispatch(
-        addReaction({
-          venueId,
-          reaction: createEmojiReaction(emojiReaction, userWithId),
-        })
-      );
-
-      // @debt Why do we have this here..? We probably shouldn't have it/need it? It's not a very Reacty thing to do..
-      setTimeout(() => (document.activeElement as HTMLElement).blur(), 1000);
-    },
-    [venueId, userWithId, dispatch]
+  const toggleMute = useCallback(
+    () => setIsAudioEffectDisabled((state) => !state),
+    []
```

Comment:
[NB] All of this can be simplified by using our `useShowHide` hook but I'm not going to make you refactor it, just pointing out that all of this could be provided by that hook.

Reply:
left comment // @debt refactor via useShowHide hook

Reply:
fixed in 9cbbfe8f7c30d7ffb04d2dd93820d4d5add71ec4